src/Html/Abstractions/src/HtmlContentBuilder.cs

```diff
@@ -183,8 +183,7 @@ public void WriteTo(TextWriter writer, HtmlEncoder encoder)
             {
                 var entry = Entries[i];
 
-                var entryAsString = entry as string;
-                if (entryAsString != null)
+                if (entry is string entryAsString)
```

Comment:
Reviewers: I recommend mainly checking that taking VS suggestions couldn't break anything. The build will validate the new `*.txt` files.